crates/ekore/src/anomalous_dimensions/unpolarized/spacelike.rs

```diff
@@ -12,9 +12,16 @@ pub mod as1;
 pub mod as1aem1;
 pub mod as2;
 pub mod as3;
+pub mod as4;
 
 /// Compute the tower of the non-singlet anomalous dimensions.
-pub fn gamma_ns_qcd(order_qcd: usize, mode: u16, c: &mut Cache, nf: u8) -> Vec<Complex<f64>> {
+pub fn gamma_ns_qcd(
+    order_qcd: usize,
+    mode: u16,
+    c: &mut Cache,
+    nf: u8,
+    n3lo_variation: [u8; 3],
```

Comment:
I'm torn between
1. having a uniform interface, i.e. passing everywhere [u8; 7] and
2. passing only the relevant information, i.e. [u8; 3]

look also to `gamma_ns_qed` ... let's also ask to @tgiani for a second opinion ... maybe at the top level (i.e. here) I slightly prefer 1.?
for 1. we could also define some constants á là `const IDX_VARIATION_GG: usize = 0;` etc.

in any case I suggest to have the documentation top-level: for singlet see my other comment and also here we need to have an explicit spell out (be it 1. or second)

Reply:
some documentation added in 4d9dfa45 - let's postpone the decision on 1. vs. 2. to later